### DISCUSSION TITLE
Why is `layout node` not recommended?

### DISCUSSION THREAD

**Discussion by zvictor** [direnv/direnv#1303] [Q&A]
The [wiki](https://github.com/direnv/direnv/wiki/Node#using-the-layout-node-directive) discourages the use of `layout node` but does not elaborate on that on any way.

> You can optionally add the `layout node` directive in your `.envrc` to have the `node_modules/bin` path added to the PATH (though, this is generally not recommended).

Could someone enlighten newcomers like me on what's wrong with that?

**Comment by zimbatm** [comment]
@wilmoore added this to the wiki
